src/controllers/dashboard/general.html

```diff
@@ -80,6 +80,13 @@ <h2>${HeaderBranding}</h2>
                     </div>
                 </div>
 
+                <div class="verticalSection">
+                    <h2>${HeaderPerformance}</h2>
+                    <div class="inputContainer">
+                        <input is="emby-input" id="txtParallelImageEncodingLimit" label="${LabelParallelImageEncodingLimit}" type="number" pattern="[0-9]*" min="1" step="1" />
```

Comment:
Shouldn't min be set to 0?

> Setting this to 0 (or less) will choose a limit based on your system specs.

Reply:
I forgot to update that. Thanks for letting me know. Fixed.